packages/ethereum-utils/src/base64.rs

```diff
@@ -0,0 +1,273 @@
+use alloy_primitives::B256;
```

Comment:
This file is quite ugly. I think there is a lot of things that can be cleaned up here, but it needs some work. Tracking in https://github.com/cosmos/solidity-ibc-eureka/issues/147

Reply:
First round of potentially dealing with this file will be done in #143, final check in #147.